.env

```
@@ -0,0 +1 @@
+DATABASE_URL="./database/"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)